source/crud/tests/v2/bulkWrite-arrayFilters-clientError.yml

```diff
@@ -0,0 +1,35 @@
+runOn:
+    -
+        # arrayFilters support first introduced in 3.5.6
+        maxServerVersion: "3.5.5"
+
+data:
+    - {_id: 1, y: [{b: 3}, {b: 1}]}
+    - {_id: 2, y: [{b: 0}, {b: 1}]}
+
+collection_name: &collection_name "test"
+database_name: &database_name "crud-tests"
+
+tests:
+  -
+    description: "BulkWrite on server (that doesn't support arrayFilters) with arrayFilters on second op"
+    operations:
+      -
+        name: "bulkWrite"
+        arguments:
+          requests:
+            -
+              # UpdateOne with no arrayFilters
+              name: "updateOne"
+              arguments:
+                filter: {}
+                update: { $set: { "y.0.b": 2 } }
+            -
+              # UpdateMany with arrayFilters
+              name: "updateMany"
+              arguments:
+                filter: {}
+                update: { $set: { "y.$[i].b": 2 } }
+                arrayFilters: [ { "i.b": 1 } ]
+          options: { ordered: true }
+          error: true
```

Comment:
Since this is asserting a client-side error, I think the test can also use `expectations: []` (as other client tests), which would assert that no command is ever sent to the server.

Reply:
Quite right.  I forgot to remove them when I trimmed down the file.  They are now gone.

Reply:
I was not aware of that.  I've added the `expectations: []` as you have suggested.